collect_app/src/androidTest/java/org/odk/collect/android/feature/settings/ResetProjectTest.kt

```diff
@@ -41,16 +40,12 @@ class ResetProjectTest {
     }
 
     @Test
-    fun canResetSavedFormsAndEntities() {
-        testDependencies.server.addForm("one-question-entity-registration.xml")
-        testDependencies.server.addForm(
-            "one-question-entity-update.xml",
-            listOf(EntityListItem("people.csv"))
-        )
+    fun canResetSavedForms() {
```

Comment:
Don't you think it would be good to have a test to make sure saved forms with entities can't be deleted here? We have one in `LocalInstanceUseCasesTest` but it is not guaranteed anywhere that `LocalInstanceUseCases` is used in `InstancesDataService`. You can remove it (or call `deleteAll` like before) and all tests will pass.

Reply:
Yeah I agree. I generally feel like a combination of feature level tests and focused unit tests gets us pretty good security around regressions, but I think you're right here that the simplest code to write (`deleteAll`) would be a really easy mistake to make and wouldn't get caught by tests.

How about I just revert [edc8e78ee033a7f8fa17721d2834955a17eb3046](https://github.com/getodk/collect/pull/6460/commits/edc8e78ee033a7f8fa17721d2834955a17eb3046)?

Reply:
Sounds good.